pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -20,6 +20,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Button, FormGroup, HelpBlock, Modal, OverlayTrigger, Tooltip, TypeAheadSelect } from 'patternfly-react';
+import { Expandable } from '@patternfly/react-core';
```

Comment:
commit message: "This option will only we available" → "be"

Reply:
Done